one more `,` in parameters

Happened in appconfiguration test

Generated tests return :
```
result = self.mgmt_client.configuration_stores.list_key_value(resource_group.name, CONFIGURATION_STORE_NAME, , BODY)

# ConfigurationStores_ListKeys[post]
BODY = {}
result = self.mgmt_client.configuration_stores.list_keys(resource_group.name, CONFIGURATION_STORE_NAME, , BODY)

# ConfigurationStores_Update[patch]
BODY = {
  "tags": {
    "category": "Marketing"
  },
  "sku": {
    "name": "Standard"
  }
}
result = self.mgmt_client.configuration_stores.update(resource_group.name, CONFIGURATION_STORE_NAME, BODY)
```

`, ,` is error

error message:
```
______ ERROR collecting sdk/appconfiguration/azure-mgmt-appconfiguration/tests/test_cli_mgmt_appconfiguration.py _______.tox/whl/lib/python3.6/site-packages/_pytest/python.py:493: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
.tox/whl/lib/python3.6/site-packages/py/_path/local.py:701: in pyimport
    __import__(modname)
<frozen importlib._bootstrap>:971: in _find_and_load
    ???
<frozen importlib._bootstrap>:955: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:665: in _load_unlocked
    ???
.tox/whl/lib/python3.6/site-packages/_pytest/assertion/rewrite.py:134: in exec_module
    source_stat, co = _rewrite_test(fn, self.config)
.tox/whl/lib/python3.6/site-packages/_pytest/assertion/rewrite.py:319: in _rewrite_test
    tree = ast.parse(source, filename=fn)
/usr/lib/python3.6/ast.py:35: in parse
    return compile(source, filename, mode, PyCF_ONLY_AST)
E     File "/_/azure-sdk-for-python/sdk/appconfiguration/azure-mgmt-appconfiguration/tests/test_cli_mgmt_appconfiguration.py", line 106
E       result = self.mgmt_client.configuration_stores.regenerate_key(resource_group.name, CONFIGURATION_STORE_NAME, , BODY)
E                                                                                                                    ^  E   SyntaxError: invalid syntax
```

--- a/src/IntegrationTest/TemplatePythonIntegrationTest.ts
+++ b/src/IntegrationTest/TemplatePythonIntegrationTest.ts
@@ -99,8 +99,19 @@ export function GeneratePythonIntegrationTest(model: Example[],
             }
         }
 
+        // format params like: xxx, xxx, xxx
+        let clientParams = _UrlToParameters(example.Url);
+        if (clientParams && hasBody)
+        {
+            clientParams += ', BODY';
+        }
+        else if (hasBody)
+        {
+            clientParams = 'BODY';
+        }
+
         output.push("        result = self.mgmt_client." + ToSnakeCase(example.OperationName) + "." + ToSnakeCase(example.MethodName) +
-                                         "(" + _UrlToParameters(example.Url) + (hasBody ? ", BODY" : "") + ")");
+                                         "(" + clientParams + ")");
         if (example.LongRunning)
         {
             output.push("        result = result.result()");
@@ -112,6 +123,7 @@ export function GeneratePythonIntegrationTest(model: Example[],
     output.push("#------------------------------------------------------------------------------");
     output.push("if __name__ == '__main__':");
     output.push("    unittest.main()");
+    output.push("");  // python pep8: need a blank line in the end.
 
     return output;
 }
@@ -181,12 +193,11 @@ function GetExampleBodyJson(body: any): string[]
 function _UrlToParameters(sourceUrl: string): string
 {
     var parts: string[] = sourceUrl.split("/");
-    var params = "";
+    var params = [];
 
     for (var i: number = 0; i < parts.length; i++)
     {
         var part: string = parts[i];
-        var last: boolean = (i == parts.length - 1);
 
         if (part.startsWith("{{"))
         {
@@ -196,16 +207,15 @@ function _UrlToParameters(sourceUrl: string): string
 
             if (varName == "RESOURCE_GROUP")
             {
-                params += "resource_group.name" + (last ? "" : ", ");
+                params.push('resource_group.name');
             }
             else
             {
-                // close and reopen quotes, add add variable name in between
-                params += varName + (last ? "" : ", ");
+                params.push(varName);
             }
         }
     }
-    return params;
+    return params.join(', ');
 }
 
 function _PythonizeBody(body: any): any

